api/proxy/store/generated_key/memstore/README.md

```diff
@@ -48,6 +48,17 @@ $ curl http://localhost:3100/memstore/config | \
   curl -X PATCH http://localhost:3100/memstore/config -d @-
 ```
 
+#### PUT with GET returning derivation error
+The configuration allows users to configure memstore to inject specific derivation error responses during GET operations while allowing PUT operations to succeed normally. This enables testing client handling of derivation errors without requiring complex setup.
+Specifically, users send a PATCH request that sets the desired derivation error for all subsequent GET requests. After that, when the user sends data to the proxy, the PUT operation succeeds as normal—the error injection only affects the GET path. Behind the scenes, upon a GET request, the proxy returns either the stored data or the specified derivation error depending on its configuration.
```

Comment:
hmm are there any unintended side effects with marking all `/get` requests (global) as a specific derivation error vs having it as a localized mutation per ephemeral db key => value entry. e.g, we disperse a payload through proxy and pre-mark it with a specific `DerivationError` vs having a global config. in the wild this error will come up  with some inbox sequence like:
```
[valid_cert, invalid_cert, valid_cert]
```

Reply:
nvm - just saw the ephemeral db changes and realized its unique to each ephemeral entry (cert, blob). Can we make this more explicit in the docs?